### PR TITLE
latest

### PR DESCRIPTION
**Pull request by ismail-ilm66**

### CODE REVIEW

File: lib/controller/location_controller.dart [hunk 1]
```diff
@@ -1,63 +1,95 @@
...
+  Rx<Position?> currentPosition = Rx<Position?>(null);
+  Rx<Set<Marker>> markers = Rx<Set<Marker>>(Set());
+  //Rx<Set<Circle>> circles = Rx<Set<Circle>>(Set());
+  GoogleMapController? mapController;
+  BitmapDescriptor? customIcon;
+
+  @override
+  void onInit() {
+    super.onInit();
+    _loadCustomIcon();
```
Comment (ismail-ilm66) [inline lib/controller/location_controller.dart:18]: there should be await here

File: lib/view/home_screen.dart [hunk 2]
```diff
@@ -14,39 +17,81 @@ class HomeScreen extends StatefulWidget {
...
+                      ),
+                      markers: locationController.markers.value,
+                      myLocationEnabled: false,
+                      //circles: locationController.circles.value,
+                      zoomControlsEnabled: false,
+                      myLocationButtonEnabled: false,
+                    ),
+                  ),
+                  Positioned(
+                    top: 30,
```
Comment (ismail-ilm66) [inline lib/view/home_screen.dart:56]: use screen utils here for the responsivness..

File: lib/view/home_screen.dart [hunk 3]
```diff
@@ -14,39 +17,81 @@ class HomeScreen extends StatefulWidget {
...
+                    child: FloatingActionButton(
+                      onPressed: () {
+                        _scaffoldKey.currentState?.openDrawer();
+                      },
+                      backgroundColor: Colors.white,
+                      shape: const CircleBorder(), // Ensure button is circular
+                      mini: true,
+                      child: const Icon(
+                        Icons.menu,
+                        size: 25,
```
Comment (ismail-ilm66) [inline lib/view/home_screen.dart:67]: same use screenutils

File: lib/view/home_screen.dart [hunk 4]
```diff
@@ -14,39 +17,81 @@ class HomeScreen extends StatefulWidget {
...
+                    right: 20,
+                    child: FloatingActionButton(
+                      backgroundColor: Colors.white,
+                      onPressed: () {
+                        locationController.moveToCurrentLocation();
+                      },
+                      shape: const CircleBorder(),
+                      child: Center(
+                        child: Image.asset(
+                          height: 25,
```
Comment (ismail-ilm66) [inline lib/view/home_screen.dart:83]: use screen utils

File: lib/controller/location_controller.dart (hunk 1 above)
Comment (MAhmad329) [reply lib/controller/location_controller.dart:18]: done

File: lib/view/home_screen.dart (hunk 2 above)
Comment (MAhmad329) [reply lib/view/home_screen.dart:56]: resolved

File: lib/view/home_screen.dart (hunk 3 above)
Comment (MAhmad329) [reply lib/view/home_screen.dart:67]: resolved

File: lib/view/home_screen.dart (hunk 4 above)
Comment (MAhmad329) [reply lib/view/home_screen.dart:83]: resolved

**Pull request closed without merging**